docs/blog/v5-intro.md

```diff
@@ -0,0 +1,167 @@
+
+### Grafana ini
+
+Just like deployment config the same thing goes for grafana config, instead of having a CRD that contains all the grafana ini values we have instead made it possible to write any value as you want.
+The **bad** thing is of course that you as an end user don't get any help with verifying types etc.
+The **good** thing is that you can set any value you want, if you want to try out an experimental feature you can do so without first having to create a PR to the operator.
+
+### Usage of status fields
+
+The usability of v4 have more then once been laking. It's been really hard to understand why something is happening and you more or less have been forces to go in and look at the logs.
+This is something that we wanted to change with v5.
+
+In v5 you will be able to look at your grafana dashboard CR and see that it got a matching grafana instance (the label selector is correct) and if it's been applied to the grafana instances.
+
+### No more plugin sidecar
+
+As you might know we have had a python container that have been an [sidecar](https://github.com/grafana-operator/grafana_plugins_init) to the operator that have been in charge of installing the plugins to grafana.
+The sidecar have been badly maintained and to be honest it's just not needed. Instead we use an environment variable to install plugins during startup.
+
+### Helm chart
+
+We have decided to start to manage our own Helm chart. The main reason for this is that we want a simple way of giving you test versions of the operator and many of you use Helm to install the operator.
+
+We still see a big place for the [community Helm chart](https://bitnami.com/stack/grafana-operator/helm) that bitnami is hosting and we hope that they will keep on doing so.
+
+### Namespace mode
+
+Another issues that we have gotten lots of feedback on is [Excessive rbac access](https://github.com/grafana-operator/grafana-operator/issues/604)
+This wasn't something that we could easily fix in v4 but in v5 we have really tried to accommodate this by introducing namespace mode.
+
+In namespace mode you can run the operator to only look for grafana resources in the same namespace that you are in or you can define it to look for a number of specific namespace. By default it will use cluster wide access.
+
+If you install through Helm or Kustomize you will be able to define if you should create cluster roles or only roles.
+Sadly for our Openshift users this won't be possible due to the limitations of OLM, even if you run the operator in namespace mode, which you can. OLM will still use the cluster wide rbac settings.
+
+## v5
+
+So there is still lots to do before version 5 will be released.
+
+There are still a number of features that is missing and we have tried to create issues around this.
+Please have a look in our v5 [milestone](https://github.com/grafana-operator/grafana-operator/milestone/1) to see all open issues.
```

Comment:
This mileston don't exist, I hope I will have time to set this up later today.
I will start moving issues from the experimental repo to the grafana-operator one and from there I will label issues and add them to milestones.